cross/Makefile

```diff
@@ -12,16 +12,22 @@ CARGO_PARAMS = --no-default-features --features bls12_377
 
 all: android ios
 
+android-setup:
+	rustup target add {aarch64-linux-android,armv7-linux-androideabi,arm-linux-androideabi,i686-linux-android,x86_64-linux-android}
```

Comment:
Interesting, seems like circleci doesn't do the shell expansion of this command. I guess we can just separate to multiple commands.